website/docs/Migration-Guide.md

```diff
@@ -3,7 +3,7 @@
 ## Migrating to 0.2
 
 openai v1 is a total rewrite of the library with many breaking changes. For example, the inference requires instantiating a client, instead of using a global class method.
-Therefore, some changes are required for users of `pyautogen<0.2`.
+Therefore, some changes are required for users of `autogen<0.2`.
```

Comment:
```suggestion
Therefore, some changes are required for users of `pyautogen<0.2`.
```
This sentence is only relevant for pyautogen users.
We probably should add a section "migration guide to 0.3" and talk about the package name change and the future plan for retiring pyautogen.